cg/cli/post_process/utils.py

```diff
@@ -1,13 +1,24 @@
+from pydantic import BaseModel, ConfigDict
+
 from cg.exc import CgError
 from cg.models.cg_config import CGConfig
+from cg.services.run_devices.abstract_classes import PostProcessingService
 from cg.services.run_devices.pacbio.post_processing_service import PacBioPostProcessingService
+from cg.services.run_devices.run_names.service import RunNamesService
 from cg.utils.mapping import get_item_by_pattern_in_source
 
 PATTERN_TO_DEVICE_MAP: dict[str, str] = {
     r"^r\d+_\d+_\d+/(1|2)_[^/]+$": "pacbio",
 }
 
 
+class UnprocessedRunInfo(BaseModel):
+    name: str
+    post_processing_service: PostProcessingService
+    instrument: str
+    model_config = ConfigDict(arbitrary_types_allowed=True)
```

Comment:
One of the instances where one could question whether it should be a base model. I am fine with either choice though.